### ISSUE TITLE
Missing data files?

### ISSUE THREAD

**Issue by Mixpap** [Santiastro1/Athens_School_2022#5]
Hi again,
I cannot find from where to download the `/EDR3_anticenter/files/edr3int4/` files. 
Thanks!

**Comment by Santiastro1** [comment]
Yes sure, this was just an "example file". In this notebook the students will need to include the file download from the Gaia data repository in the "csv" format.
I think it can be just an "extra" exercise. One hour is small time to go through the Gaia data exercise + download the data from DR3.
Next week I will work to produce a small example file to be included in this notebook.
